sdccc/src/main/java/com/draeger/medical/sdccc/messages/MessageStorage.java

```diff
@@ -444,11 +455,15 @@ private void checkMimeType(final Message message) {
                     mimeType = value;
                 }
                 if (!SDC_MIME_TYPES.contains(mimeType)) {
-                    this.testRunObserver.invalidateTestRun(String.format(
-                            "encountered a SOAP Envelope whose mimeType '%s' (declared in its "
-                                    + "HTTP Header) indicates that it was not serialized as 'application/soap+xml' and "
-                                    + "that hence violates the definition of a SOAP TEXT ENVELOPE in MDPWS Section 3.1.",
-                            mimeType));
+                    if (summarizeMessageEncodingErrors) {
```

Comment:
The test run will not be invalidated if SummarizeMessageEncodingErrors is set.

Reply:
Redesigned it. Now there is a separate Configuration Key for that.